content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -106,29 +108,64 @@ when the Pod is created, so it is ignored by the scheduler).  Therefore:
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
-Daemon Pods do respect [taints and tolerations](/docs/concepts/configuration/taint-and-toleration),
-but they are created with `NoExecute` tolerations for the following taints with no `tolerationSeconds`:
+### Scheduled by default scheduler (with ScheduleDaemonSetPods alpha feature)
+
+A DaemonSet ensures that all (or some) nodes run a copy of a pod. Normally, the machine that a pod runs on is selected by the Kubernetes scheduler; however, pods of DaemonSet are created and scheduled by DaemonSet controller who leveraged kube-scheduler’s predicates policy. That introduces the following issues:
```

Comment:
Remove "who leveraged kube-scheduler’s predicates policy"; this is implementation details.